packages/KTable/KTable.vue

```diff
@@ -164,6 +171,13 @@ export default {
     sortKey: {
       type: String,
       default: ''
+    },
+    /**
+     * A function that conditionally specifies the row classes on the rows
+     */
+    rowClasses: {
```

Comment:
I think it would make our lives easier if we v-bind on `tr` and call this `rowAttributes`. For example, if a user wants to add anything e.g. `data-testid` to the rows then we wouldn't need to add a new property.